account_invoice_batches/__manifest__.py

```diff
@@ -14,6 +14,7 @@
         "l10n_es_facturae",
         "account_invoice_report_service",
         "sale_order_invoicing_grouping_criteria",
+        "account_mail_attach_existing_attachment",
```

Comment:
Please, do the same without this dependency, It should be possible to have installed one module without the other.